<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateExpDialog</class>
 <widget class="QDialog" name="CreateExpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create EXP from RUN</string>
  </property>
  <widget class="QPushButton" name="AddMeasPushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Considering all the imported RUNs, apply the below filters and add files to EXP</string>
   </property>
   <property name="text">
    <string>Add Measurements to EXP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FilterMeasPushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Considering the files already in the EXP, keep the files that meet all criteria</string>
   </property>
   <property name="text">
    <string>Filter Measurments in EXP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ImportRunsPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Import multiple folders and/or .zip, each representing a RUN with a .rcp file</string>
   </property>
   <property name="text">
    <string>Import RUN(s)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FilterRunComboBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Apply all other filteres in this section to only this run</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PlateAttrEqualComboBox">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>30</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>These are attributes from the platemap file and select values or ranges can be filtered with these 3 (in)equalities</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>30</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PlateAttrEqualSpinBox">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PlateAttrLessSpinBox">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>60</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="PlateAttrLessComboBox">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>60</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>60</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PlateAttrMoreSpinBox">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>90</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>90</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PlateAttrMoreComboBox">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Require the Sample properties:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Only consider this RUN</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ExpTextBrowser">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>531</width>
     <height>381</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveExpPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>save .exp text file</string>
   </property>
   <property name="text">
    <string>Save EXP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ImportExpPushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Import a .exp file, which will clear any existing EXP and RUNs and create new RUNs from the EXP</string>
   </property>
   <property name="text">
    <string>Import EXP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileSearchLineEdit">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>220</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>All raw data filenames regardless of type will be searched for this string.
 comma-deliminated strings will be searched separately with OR logic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>200</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Require filename to contain:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SampleListLineEdit">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>140</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>This should be a single sample number of comma-deliminated list of numbers from the platemap file</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>120</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample list, e.g. 3,77,111</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ExpDescLineEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>400</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Comment string to be included in EXP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EXP description:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RunPriorityLineEdit">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>260</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>comma-deliminated list of RUN indeces in tree view.
If the same filename appears in multiple runs, this sets the priorty for which is used.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>240</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RUN priority (high to low):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RunTypeLineEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>When &quot;Add&quot; to EXP the files within a RUN will be marked with this type</string>
   </property>
   <property name="text">
    <string>data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RUN type:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveRunsPushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Import multiple folders and/or .zip, each representing a RUN with a .rcp file</string>
   </property>
   <property name="text">
    <string>Remove all RUNs</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LastActionLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Comment string to be included in EXP</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Action:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ExpTypeLineEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Comment string to be included in EXP</string>
   </property>
   <property name="text">
    <string>eche</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EXP type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>340</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EXP name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ExpNameLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>340</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Comment string to be included in EXP</string>
   </property>
   <property name="text">
    <string>eche</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>370</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>created by:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserNameLineEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>370</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Comment string to be included in EXP</string>
   </property>
   <property name="text">
    <string>eche</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="AccessLineEdit">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>370</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Comment string to be included in EXP</string>
   </property>
   <property name="text">
    <string>hte</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>370</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EXP access:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearExpPushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>save .exp text file</string>
   </property>
   <property name="text">
    <string>Clear EXP</string>
   </property>
  </widget>
  <widget class="QComboBox" name="BatchComboBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Apply all other filteres in this section to only this run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BatchPushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>300</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Considering the files already in the EXP, keep the files that meet all criteria</string>
   </property>
   <property name="text">
    <string>Run Batch Process:</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>280</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ImportExpParamsPushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Open a .exp file and use only the top-level params</string>
   </property>
   <property name="text">
    <string>Import EXP Params</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchRunsPushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Import multiple folders and/or .zip, each representing a RUN with a .rcp file</string>
   </property>
   <property name="text">
    <string>Search for RUNs</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="RunTreeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>491</width>
     <height>351</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
   <property name="expandsOnDoubleClick">
    <bool>false</bool>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QTreeWidget" name="ExpTreeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>491</width>
     <height>351</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
   <property name="expandsOnDoubleClick">
    <bool>false</bool>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>160</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Require filename start with:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileStartLineEdit">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>180</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>All raw data filenames regardless of type will be searched for this string.
 comma-deliminated strings will be searched separately with OR logic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveExpGoAnaPushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>save .exp text file</string>
   </property>
   <property name="text">
    <string>Save+GoToAna</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="getplatemapCheckBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>119</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get platemaps
on RUN import</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserFOMLineEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>enter comma-delimited list of string or
number FOMS that will become a constant column in the .csv generated by &quot;Analyze Data&quot;.
After entry complete, you will be prompted for fom names</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User-defined FOMs</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="TechTypeTreeWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>171</width>
     <height>251</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
   <property name="expandsOnDoubleClick">
    <bool>false</bool>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select tech+type to add/keep:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Run(s) description for next &quot;add&quot;:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RunDescLineEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>enter comma-delimited list of string or
number FOMS that will become a constant column in the .csv generated by &quot;Analyze Data&quot;.
After entry complete, you will be prompted for fom names</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
